core/blockly.js

```diff
@@ -661,3 +668,13 @@ Blockly.checkBlockColourConstant_ = function(
     console.warn(warning);
   }
 };
+
+/**
+ * Set the parent container.  This is the container element that the WidgetDiv,
+ * DropDownDiv, and Tooltip are rendered into the first time `Blockly.inject`
+ * is called.
```

Comment:
Add a comment: this is a no-op if called after the first call to `inject`.

Reply:
Done.